discussion_agents/cog/agent/react.py

```diff
@@ -20,11 +20,28 @@
 from langchain_core.tools import BaseTool, tool
 from tiktoken.core import Encoding
 
+import yaml
+import alfworld
+import alfworld.agents.environment
+
+
 from discussion_agents.cog.agent.base import BaseAgent
-from discussion_agents.cog.functional.react import _is_halted, _prompt_agent
+from discussion_agents.cog.functional.react import _is_halted, _prompt_agent , _check_keyword , _process_ob
 from discussion_agents.cog.modules.memory.react import ReActMemory
 from discussion_agents.utils.parse import parse_action, remove_newline
 
+from discussion_agents.cog.prompts.react import (
+    REACT_INSTRUCTION_HOTPOTQA,
+    REACT_INSTRUCTION_FEVER,
+    REACT_ALFWORLD_INSTRUCTION
+)
+
+import yaml
+import alfworld
+import alfworld.agents.environment
```

Comment:
these imports should be above library imports (discussion agents). generally split imports into 2 groups: general and specific. specific imports are from our library and general are from standard python libs or third party libs.